tasks/test_jest.js

```diff
@@ -18,15 +18,13 @@ module.exports = function (grunt) {
     return new Promise((resolve, reject) => {
       grunt.util.spawn(serverCmd, (error, result, code) => {
         if (error || code !== 0) {
-          const message = result.stderr || result.stdout;
-
-          grunt.log.error(message);
-
-          return reject();
+          const error = new Error(`jest exitted with code ${code}`);
```

Comment:
I know making up words is "cool" and "hip", but the past tense of "exit" is "exited" according to the googles.